src/components/status-chip/StatusChip.styles.ts

```diff
@@ -5,12 +5,14 @@ import { StatusEnum } from '~/types'
 const statusColors = {
   pending: palette.basic.blue,
   active: palette.success[600],
-  closed: palette.primary[400]
+  closed: palette.primary[400],
+  draft: palette.basic.blue
 }
 
 export const styles = (status: StatusEnum) => ({
   root: {
     backgroundColor: alpha(statusColors[status], 0.2),
+    border: `1px solid ${statusColors[status]}`,
```

Comment:
split to border and borderColor, which will be more readable than template

Reply:
fixed